lib/services/lookup.js

```diff
@@ -482,6 +482,31 @@ function lookupServiceFactory(
     };
 
     /**
+     * Returns a list of object containing IP and Mac Addresses for the given Node ID.
+     * @param  {String} id Node ID
+     * @return {Promise.<Array>} A promise fulfilled with an array of
+     * objects containing IP Addresses and Mac Addresses associated with the Node.
+     */
+    LookupService.prototype.findIpMacAddresses = function (id) {
```

Comment:
A little suggestion, change the variable name from `id` to `nodeId` will be better, since the the caller may confuse the id is the lookup document id.